### PR TITLE
port to v10

### PR DESCRIPTION
**Pull request by c-ehrlich**
ports kitchen-sink to v10, and also fixes a few other things (errors relating to deprecated tailwind stuff etc)

known issues:
- hydration error in next-auth example, this is the same in the current v9 kitchen sink
  - works after build && start, but with errors logged to console

### CODE REVIEW

File: src/feature/react-hook-form/index.tsx [hunk 1]
```diff
@@ -16,24 +16,29 @@ function useZodForm<TSchema extends z.ZodType>(
...
+
 export default function Page() {
-  const utils = trpc.useContext();
-  const query = trpc.useQuery(['reactHookForm.list'], { suspense: true });
+  const utils = useTRPCContext();
+  const query = trpc.list.useQuery(undefined, {
+    suspense: true,
+    // not refetching, because we're not actually persisting any posts on the server
+    refetchInterval: Infinity,
+    refetchOnWindowFocus: false,
```
Comment (KATT) [inline src/feature/react-hook-form/index.tsx]: For some reason, this actually works for a few seconds on Vercel
```suggestion
```

**Review by KATT — COMMENTED** [review]
Looks very good - I've switched so `next` is the default branch but cool to keep this open to see differences :)

**Comment by c-ehrlich** [comment]
probably won't have time to take another look at the hydration stuff until this weekend at the earliest.

but it behaved the way in the v9 example so i could also see just using this example as it is for now, and opening an issue about the hydration error either here or on the main trpc repo.

**Comment by KATT** [comment]
> probably won't have time to take another look at the hydration stuff until this weekend at the earliest.
> 
> but it behaved the way in the v9 example so i could also see just using this example as it is for now, and opening an issue about the hydration error either here or on the main trpc repo.

Fixed in ab1fd63

**Pull request closed without merging**